### PR TITLE
Fix archiving

### PR DESCRIPTION
**Pull request by vanmaxim**

### CODE REVIEW

File: MapboxVision/Services/Recording/RecordArchiver.swift [hunk 1]
```diff
@@ -10,14 +10,12 @@ import Foundation
...
 }
 
 final class RecordArchiver: Archiver {
     
-    func archive(_ files: [URL], destination: URL) throws {
-        for file in files {
-            try FileManager.default.zipItem(at: file, to: destination)
-        }
+    func archive(_ folder: URL, destination: URL) throws {
+            try FileManager.default.zipItem(at: folder, to: destination)
```
Comment (chezzdev) [inline MapboxVision/Services/Recording/RecordArchiver.swift]: Indentation if off

**Comment by vanmaxim** [comment]
@chezzdev PTAL

File: MapboxVision/Services/Recording/RecordArchiver.swift [hunk 2]
```diff
@@ -15,9 +15,17 @@ protocol Archiver {
...
+    }
+    
     func archive(_ files: [URL], destination: URL) throws {
+        guard let archive = Archive(url: destination, accessMode: .create) else {
+            throw RecordArchiverError.cantCreateArchive(destination)
+        }
+        
         for file in files {
-            try FileManager.default.zipItem(at: file, to: destination)
+            try archive.addEntry(with: file.lastPathComponent, relativeTo: file.deletingLastPathComponent())
```
Comment (chezzdev) [inline MapboxVision/Services/Recording/RecordArchiver.swift]: Please check that `relativeTo` doesn't create extra directories.

Comment (vanmaxim) [reply MapboxVision/Services/Recording/RecordArchiver.swift]: Checked

**Pull request merged**